core/pom.xml

```diff
@@ -847,7 +847,7 @@ THE SOFTWARE.
           <plugin>
             <groupId>com.github.siom79.japicmp</groupId>
             <artifactId>japicmp-maven-plugin</artifactId>
-            <version>0.14.4-20200728.214757-1</version> <!-- TODO https://github.com/siom79/japicmp/pull/266 -->
+            <version>0.15.2</version> <!-- TODO https://github.com/siom79/japicmp/pull/266 -->
```

Comment:
PR from @jglick has not been merged yet

Reply:
https://github.com/siom79/japicmp/pull/266 yes. We can use japicmp without it for routine API changes, but the comparison would fail in cases of classpath changes such as #5059.